File: Lambda/Watch_Next_By_Id/Talk.js
const mongoose = require('mongoose');

const talk_schema = new mongoose.Schema({
    _id: String,
    title: String,
    url: String,
    description: String,
    speakers: String,
    watch_next: Array
}, { collection: 'tedx_data' });

module.exports = mongoose.model('talk', talk_schema);

File: Flutter/my_teducation/lib/models/talk.dart
class Talk {
  final String id;
  final String title;
  final String details;
  final String mainSpeaker;
  final String url;
  final String views;
  final String imageurl;
  final List watchnext;



  Talk.fromJSON(Map<String, dynamic> jsonMap) :
    id = jsonMap['internalId'],
    title = jsonMap['title'],
    details = jsonMap['description'],
    mainSpeaker = (jsonMap['speakers'] ?? ""),
    url = (jsonMap['url'] ?? ""),
    views = (jsonMap['views'] ?? "0"),
    imageurl = (jsonMap['image_url'] ?? ""),
    watchnext = (jsonMap['watch_next'] ?? "");
}